nuget/readme.md

```diff
@@ -109,3 +109,118 @@ When you publish to GitHub the lib directory should contain nuget.config file wi
     </packageSourceCredentials>
 </configuration>
 ```
+
+# Build Bling Fire dlls/NuGet package with Azure DevOps build pipeline
+
+Now we can use Azure DevOps to build BlingFire dlls and NuGet package.
+
+Pipeline file [azure-pipelines.yml](../azure-pipelines.yml) is now updated to be able to build BlingFire dlls for windows-x64, windows-arm64, osx-x64, osx-arm-64, linux-x64, linux-arm64.
+
+Here are some good documentations about how to create Azure DevOps pipelines:
+
+1. [Create your first pipeline.](https://docs.microsoft.com/en-us/azure/devops/pipelines/create-first-pipeline?view=azure-devops&tabs=java%2Ctfs-2018-2%2Cbrowser)
+2. [Build GitHub repositories.](https://docs.microsoft.com/en-us/azure/devops/pipelines/repos/github?view=azure-devops&tabs=yaml)
+
+A good thing about using Azure DevOps is: it provides [list of Microsoft-hosted agents](https://docs.microsoft.com/en-us/azure/devops/pipelines/agents/hosted?view=azure-devops&tabs=yaml), which include Windows, Linux and Osx, with all the popular tools pre-installed.
+
+However, there are no ARM64 microsoft-hosted agents yet, so currently in [azure-pipelines.yml](../azure-pipelines.yml) all the ARM64 dlls are built with cmake cross-compiling from X64 agents.
+<br/><br/>
+
+## Build pipeline for Github BlingFire repo 
+A [build pipeline](https://devdiv.visualstudio.com/DevDiv/_build?definitionId=16503&_a=summary) for [Github BlingFire repo](https://github.com/microsoft/BlingFire) is created (only accessble by Microsoft interal users). It will be triggered whenerver thare's new commeits to master branch.

```

Comment:
This pipeline is actually against my forked repo, I don't have enough access to create pipeline against the original repo.